app/components/go_component.rb

```diff
@@ -0,0 +1,45 @@
+class GoComponent < ViewComponent::Base
+  include Motion::Component
+
+  map_motion :place
+  map_motion :reset
+
+  def initialize(key:)
+    @key = key
+    @game = GoGame.find(key: key) || GoGame.new(key: key)
+    puts "component game #{@game}"
+    @board = @game.display
+    @current = @game.current
+    @captures = @game.captures
+
+    stream_from "go:#{@key}", :player_move
+  end
+
+  def place(event)
+    index = event.target.data[:index].to_i
+    pos = GoGame::Pos.new((index / 9), index.modulo(9))
+
+    if @game.legal_move?(pos)
+      @game.place(pos)
+      @game.next_player
+      GoGame.update(key: @key, game: @game)
+    end
+  end
+
+  def reset
```

Comment:
I think this functionality isn't built out.

One reason we might not want this yet is that the url will stay the same but the game will be different, so a player wanting to invite another player to a new game will not be able to (I think)?

Reply:
yeah, this is vestigial. i forgot to remove this after moving logic to support multiple games with ids instead of just one.